src/test/scala-2.11/markets/participants/TestLiquiditySupplier.scala

```diff
@@ -18,32 +18,29 @@ package markets.participants
 import akka.actor.{Props, ActorRef}
 import akka.agent.Agent
 
-import markets.orders.Order
-import markets.orders.limit.{LimitAskOrder, LimitBidOrder}
+import markets.participants.strategies.TestLimitOrderTradingStrategy
 import markets.tickers.Tick
 import markets.tradables.Tradable
 
 import scala.collection.mutable
-import scala.util.Random
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705065123/issues/source?bid=2921361&fileBranchId=3046267#l27)